scgallery/designs/__init__.py

```
@@ -27,62 +29,35 @@
+    "MockALUDesign",
+    "OpenMSP430Design",
+    "PicoRV32Design",
+    "QERVDesign",
+    "Riscv32iDesign",
+    "SERVDesign",
+    "SPIDesign",
+    "SwervDesign",
+    "TinyRocketDesign",
+    "UARTDesign",
+    "WallyDesign"
 ]
```

Ruff (0.14.0)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)